megalut/sim/stampgrid.py

```diff
@@ -159,6 +159,8 @@ def drawimg(catalog, simgalimgfilepath="test.fits", simtrugalimgfilepath=None, s
 	"""
 	starttime = datetime.now()	
 	
+	gsparams = galsim.GSParams(maximum_fft_size=8192)
```

Comment:
Yes, from time to time. Default is 4096, this is just twice. We don't have another mecanism to specify this option, so I just hard-coded it, guessing that we all have machines which are large enough.